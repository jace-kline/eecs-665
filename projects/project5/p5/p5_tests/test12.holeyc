void f() {
    return;
}

int g(bool b) {
    return;
}

void main() {
    TOCONSOLE f;
    TOCONSOLE f();
    return 5;
}