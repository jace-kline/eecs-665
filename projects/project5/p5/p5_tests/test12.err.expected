FATAL [6,5]: Missing return value
FATAL [10,15]: Attempt to output a function
FATAL [11,15]: Attempt to write void
FATAL [12,12]: Return with a value in void function
Type Analysis Failed
